lua/atone/tree.lua

```diff
@@ -76,22 +135,28 @@ function M.convert(buf)
         -- child is a descendant with the same depth as the node.
         child = nil,
         children = {},
+        bufnr = buf,
+        time = os.time(),
     }
     M.cur_seq = undotree.seq_cur
     M.last_seq = undotree.seq_last
     if M.last_seq == 0 then
         return M.nodes
     end
 
+    if undotree.entries[1] == nil then
```

Comment:
We don't need this condition check after https://github.com/XXiaoA/atone.nvim/commit/86b5ae456a13182b274964ebb28be78f7fcbff3c